addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -9051,42 +9051,48 @@ QUnit.module("Views", (hooks) => {
         }
     );
 
-    QUnit.skipWOWL("keep editing after call_button fail", async function (assert) {
+    QUnit.test("keep editing after call_button fail", async function (assert) {
         assert.expect(4);
 
-        var values;
+        let values;
+
+        // FIXME: there should be no depencency from views to actionService
+        const mockedActionService = {
+            start() {
+                return {
+                    doActionButton() {
+                        assert.ok(true, "the action is correctly executed");
```

Comment:
maybe check the params (type and name), it's easy and looks a bit better than assert.ok(true)